nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/RHostedIT.java

```diff
@@ -124,84 +113,85 @@ public void testFetchNotExistingPackage() throws Exception
     assertThat(client.fetch(AGRICOLAE_131_TARGZ.fullPath).getStatusLine().getStatusCode(), is(NOT_FOUND));
   }
 
-  @Test
-  public void testMetadataProcessing() throws Exception
-  {
-    // Uploading package with same name and lower version that should be skipped in src metadata
-    uploadSinglePackage(AGRICOLAE_101_TARGZ);
-
-    Thread.sleep(METADATA_PROCESSING_WAIT_INTERVAL_MILLIS);
-
-    final String agricolae121Content =
-        new String(Files.readAllBytes(testData.resolveFile(PACKAGES_AGRICOLAE_121_FILENAME).toPath()));
-    final String agricolae131Content =
-        new String(Files.readAllBytes(testData.resolveFile(PACKAGES_AGRICOLAE_131_FILENAME).toPath()));
-
-    // Verify PACKAGES(metadata) contain appropriate content about source R package (version 1.0-1 is skipped)
-    final InputStream contentSrc = client.fetch(PACKAGES_SRC_GZ.fullPath).getEntity().getContent();
-    verifyTextGzipContent(is(equalTo(agricolae121Content)), contentSrc);
-    assertNotNull(findAsset(repository, PACKAGES_SRC_GZ.fullPath));
-
-    // Verify PACKAGES(metadata) contain appropriate content about bin R package
-    final InputStream contentBin = client.fetch(PACKAGES_BIN_GZ.fullPath).getEntity().getContent();
-    verifyTextGzipContent(is(equalTo(agricolae131Content)), contentBin);
-    assertNotNull(findAsset(repository, PACKAGES_BIN_GZ.fullPath));
-
-    // Verify PACKAGES(metadata) is clean if component has been deleted
-    List<Component> components = getAllComponents(repository);
-    ComponentMaintenance maintenanceFacet = repository.facet(ComponentMaintenance.class);
-    components.forEach(component -> maintenanceFacet.deleteComponent(component.getEntityMetadata().getId()));
-
-    Thread.sleep(METADATA_PROCESSING_WAIT_INTERVAL_MILLIS);
-
-    final InputStream contentSrcAfterDelete = client.fetch(PACKAGES_SRC_GZ.fullPath).getEntity().getContent();
-    verifyTextGzipContent(is(equalTo("")), contentSrcAfterDelete);
-    assertNotNull(findAsset(repository, PACKAGES_SRC_GZ.fullPath));
-
-    final InputStream contentBinAfterDelete = client.fetch(PACKAGES_BIN_GZ.fullPath).getEntity().getContent();
-    verifyTextGzipContent(is(equalTo("")), contentBinAfterDelete);
-    assertNotNull(findAsset(repository, PACKAGES_BIN_GZ.fullPath));
-  }
+ // @Test
```

Comment:
rather than commenting out the following code, an `@Ignore` would be better, IMO. You can even add a value that links back to something like a JIRA ticket for why the test is now disabled.

Reply:
@wwannemacher  agree. I also don't like commented code in the PR but this one is not going to be merged to master. it is still in progress. I going to fix it. But once I see that for some reason the test is not possible to fix I will use Ignore annotation with reference to the Ticket. 
Thanks you take care of good codebase and best practices Wes!